specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-04-01-preview/kustoPool.json

```diff
@@ -0,0 +1,1534 @@
+          "type": "string",
+          "description": "The engine type",
+          "enum": [
+            "V2",
+            "V3"
+          ],
+          "x-ms-enum": {
+            "name": "EngineType",
+            "modelAsString": true
+          }
+        },
+        "workspaceUid": {
+          "type": "string",
+          "description": "The workspace unique identifier."
+        }
+      },
+      "description": "Class representing the Kusto pool properties."
+    },
+    "KustoPool": {
+      "required": [
+        "sku"
+      ],
+      "properties": {
+        "sku": {
+          "$ref": "#/definitions/AzureSku",
+          "description": "The SKU of the kusto pool."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/KustoPoolProperties",
+          "description": "The kusto pool properties."
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        },
+        "systemData": {
+          "readOnly": true,
+          "type": "object",
+          "description": "Azure Resource Manager metadata containing createdBy and modifiedBy information.",
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Class representing a Kusto kusto pool."
+    },
+    "KustoPoolUpdate": {
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-mutability": [
+            "read",
+            "create",
+            "update"
+          ],
+          "description": "Resource tags."
+        },
+        "sku": {
+          "$ref": "#/definitions/AzureSku",
+          "description": "The SKU of the kusto pool."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "The kusto pool properties.",
+          "$ref": "#/definitions/KustoPoolProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Resource"
+        }
+      ],
+      "description": "Class representing an update to a Kusto kusto pool."
+    },
+    "DatabaseListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Database"
+          },
+          "description": "The list of Kusto databases."
+        }
+      },
+      "description": "The list Kusto databases operation response."
+    },
+    "Database": {
+      "required": [
+        "kind"
+      ],
+      "properties": {
+        "location": {
```

Comment:
Is proxy resource with location supported? @MarcinJastrzebski

Reply:
Yeah, but it doesn't change routing like it does in tracked resources.

---
In reply to: [612004189](https://github.com/Azure/azure-rest-api-specs/pull/13874#discussion_r612004189) [](ancestors = 612004189)